core-web/apps/dotcms-ui/src/app/view/components/dot-content-compare/store/dot-content-compare.store.ts

```diff
@@ -87,7 +87,28 @@ export class DotContentCompareStore extends ComponentStore<DotContentCompareStat
                                         return { contentType, contents };
                                     })
                                 );
-                        })
+                        }),
+                        switchMap(
+                            (value: {
+                                contentType: DotCMSContentType;
+                                contents: DotCMSContentlet[];
+                            }) => {
+                                return !value.contents.some(
+                                    (content) => content.inode === data.inode
+                                )
+                                    ? this.dotContentletService
```

Comment:
add a test that this method is being called in case is not part of the first results. also one line doc to remember why we did this.

Reply:
Comment line here https://github.com/dotCMS/core/pull/27663/commits/f99abaa010c409d6185f0849715c75e06bf76a2c

Reply:
We already have a test for this method here https://github.com/dotCMS/core/blob/619a5c7f3c71cf0e6e3724257c84fd183c3f7597/core-web/apps/dotcms-ui/src/app/view/components/dot-content-compare/store/dot-content-compare.store.spec.ts#L725
Since we are already testing the modified method, I think that what we need to do is increase the number of items in the content version list. When we run the current test, it should work as well.
I am adding more content here:

https://github.com/dotCMS/core/blob/619a5c7f3c71cf0e6e3724257c84fd183c3f7597/core-web/apps/dotcms-ui/src/app/view/components/dot-content-compare/store/dot-content-compare.store.spec.ts#L680

Do you think this is a good approach?